lib/ansible/modules/windows/win_domain_group_membership.ps1

```diff
@@ -52,7 +54,17 @@ $members_before = Get-AdGroupMember -Identity $name @extra_args
 $pure_members = [System.Collections.Generic.List`1[String]]@()
 
 foreach ($member in $members) {
-    $group_member = Get-ADObject -Filter "SamAccountName -eq '$member' -and $ad_object_class_filter" -Properties objectSid, sAMAccountName
+    $extra_member_args = @{}
```

Comment:
It's probably better to do `$extra_member_args.Clone()` then set `$extra_member_args.Server = $member.Split("\")[0]` if it's in the Netlogon form with the server. That way you don't have to check if Credentials is set again and you can remove the else part of the if statement.